english/super-man/save-the-world.md

```diff
@@ -0,0 +1,12 @@
+After that, suddenly!!! You start to grow up, Big, GIANT ~~~ 
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.